### DISCUSSION TITLE
Rejected my requests of GitHub Global Campus many times

### DISCUSSION THREAD

**Discussion by ghost** [community/community#56593] [GitHub Education]
### Hi everyone,

I have approved for GitHub Global Campus two years ago when I was 8th grade.
And I am trying to renew that approval with my mail address ending with `ed.jp`

However, despite sending many requests for about a month, they are rejected.
I have already sent the following documents:
- School ID card with the date of issue and the date of expiration
- Admission information with the date
- Receipt of payment for admission to the school with the date
- Certificate of enrollment in English with the date of issue and signature of the principal
- Photocopies of both sides of school ID card

I took them with the browser's camera function.

But it gets rejected like the image below:
<img width="1122" alt='Rejection mail for "common verification problems"' src="https://github.com/community/community/assets/79352785/228eb765-4db9-469b-9acc-be932688e842">

What more should I send to be approved?

I'm Japanese and not good at English, so I'm sorry if I made mistakes.

Thank you.

**Comment by Jake7208** [comment]
You could contact support, and show if they ask to see your school id if they manually check it, it may pass.

**Reply by ghost** [reply]
Thanks for your comment @Jake7208 

However, support is no longer inquiring about GitHub Education's rejection of my application. Is there any other way to contact them?

**Reply by Jake7208** [reply]
I wish I could be of more help but the only thing I can think of is create a new ticket 
https://support.github.com/request?tags=docs-generic

**Reply by ghost** [reply]
Thank you. I'll give it a try.

**Comment by samus-aran** [comment]
Hi @sera1mu 👋 

There are a few things here you can review:

1. Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub **username, email** and name ( as well as other points) to cross reference the evidence submitted. I would suggest you look at how you can confirm that the evidence submitted matches up with your GitHub account. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
2. If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
